File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/plagiarized/L5/06/TestSatuArray.kt
package IR

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    val arr = IntArray(10) // initialize array
    inputArr(arr)
    printArr(arr)
}

fun inputArr(arr: IntArray) {
    val sc = Scanner(System.`in`)
    var x = 0
    while (x < 10) { // loop for input array
        println("Read a number: ")
        arr[x] = sc.nextInt() // input to array
        x++
    }
}

fun printArr(arr: IntArray) {
    var x = 9
    while (x >= 0) { // loop for displaying array
        println(arr[x]) // display array
        x--
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L5/06/TestSatuArray.py
import sys

def inputArr(arr):
    for x in range(10):
        arr[x] = int(input("Read a number: "))
    return arr

def printArr(arr):
    for x in range(9, -1, -1):
        print(arr[x])

if __name__ == "__main__":
    arr = [0] * 10
    arr = inputArr(arr)
    printArr(arr)

